deploy/katello.spec

```diff
@@ -148,6 +161,27 @@ getent group katello-shared > /dev/null || groupadd -r katello-shared
 usermod -a -G katello-shared katello
 exit 0
 
+# ------ SAM ------------------
+
+%package sam
+Summary: Package that installs only the Subscription and basic Content Management parts of Katello
+Group:  Applications/System
+
+# Require the common package and ensure the katello-sam package 
+# can't be installed on the same system as katello
+Requires:       %{name}-common = %{version}-%{release}
+Conflicts:      %{name}
+Requires:       gutterball
+Requires:       rubygem-hammer_cli_sam
+Requires:       rubygem-hammer_cli_gutterball
+Requires:       ruby193-rubygem-foreman_sam
+Requires:       ruby193-rubygem-foreman_gutterball
```

Comment:
not all of it, but you are right, I need to put gutterball into common but the unique stuff to -sam is the sam specific packages.